src/LibraryManager/Providers/jsDelivr/JsDelivrCatalog.cs

```diff
@@ -68,12 +68,37 @@ public async Task<string> GetLatestVersion(string libraryId, bool includePreRele
                                                                                                            cancellationToken).ConfigureAwait(false);
 
                 var packageObject = (JObject)JsonConvert.DeserializeObject(latestVersionContent);
-                if (packageObject != null)
+
+                if (includePreReleases)
                 {
-                    var versions = packageObject["tags"] as JObject;
-                    var versionValue = versions["latest"] as JValue;
-                    latestVersion = versionValue?.Value as string;
+                    // there isn't a reliable tag to use for prerelease versions, so we'll have to parse the full list
+                    if (packageObject["versions"] is JArray versions)
+                    {
+                        if (versions.Values() is IEnumerable<JToken> versionValues)
+                        {
+                            latestVersion = versionValues.Select(vv => SemanticVersion.Parse(vv.ToString()))
+                                                         .Max()
+                                                         .ToString();
+                        }
+                    }
                 }
+                else
+                {
+                    if (packageObject["tags"] is JObject tags
+                        && tags["latest"] is JValue latestTag)
+                    {
+                        latestVersion = latestTag.Value as string;
+                    }
+                    else if (packageObject["versions"] is JArray versions
+                             && versions.Values() is IEnumerable<JToken> versionValues)
+                    {
+                        latestVersion = versionValues.Select(vv => SemanticVersion.Parse(vv.ToString()))
+                                                     .Where(semanticVersion => semanticVersion.PrereleaseVersion == null)
+                                                     .Max()
+                                                     .ToString();
+                    }
```

Comment:
Was this `else if` part added because the `latest` tag isn't reliable either (similar to your comment above about prerelease? Might be worth a comment for why the fallback is needed.

Reply:
It was primarily added because the GitHub versions doesn't include a "latest" tag (actually, it doesn't include any tags from my poking about).  I'm not sure whether it's also possible that they would have an NPM package that didn't have "latest" since there isn't consistency in the pre-release tags, so catching that is an extra bonus, but it's mostly driven to support the GH versions.